src/data.js

```diff
@@ -316,9 +317,12 @@ Artifact.prototype.remove = function(ignoreError) {
     } else if (this.details.bucket === this.privateBucket.bucket) {
       deleted = this.privateBucket.deleteObject(this.details.prefix);
     } else {
-      debug("[alert-operator] Expiring artifact with bucket: %s, which isn't " +
-            "configured for use. Please investigate taskId: %s, runId: %s",
-            this.details.bucket, this.taskId, this.runId);
+      let err = new Error("Expiring artifact with bucket which isn't " +
```

Comment:
nit: you intermingle single and double quotes when you deal with strings, I suppose the linter stuff will pick up on this if there is the right rule for it.